1.4Tablice/zad1/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿Console.Write("Podaj wielkość tablicy: ");
+var size = int.Parse(Console.ReadLine());
+
+var array = new int[size];
+for (var i = 0; i < size; i++)
+{
+    array[i] = int.Parse(Console.ReadLine());
+}
+
+Console.WriteLine("Uporządkowanie elementów w tablicy od najmniejszego do największego i wypisanie ich - 1");
+Console.WriteLine("Uporządkowanie elementów tablicy od największego do najmniejszego i wypisanie ich - 2");
+Console.WriteLine("Wypisanie największego i najmniejszego elementu - 3");
+Console.WriteLine("Wypisanie elementu o podanym nr indexu (po wybraniu tej opcji aplikacja wypiszę dostępne indexy elementów) - 4");
+Console.WriteLine("Zastąpienie elementu o podanym indexie, nowym elementem - 5");
+var taskNumber = int.Parse(Console.ReadLine());
+
+if (taskNumber == 1)
+{
+    var newArray = array.OrderBy(x => x).ToArray();
+    WriteArray(newArray);
```

Comment:
FIX: Jak uwzględnisz komentarz dotyczący metody `WriteArray` to możesz to zapisać:
```suggestion
    WriteArray(array.OrderBy(x => x));
```

Reply:
[181d2f9](https://github.com/broniuu/Wst-pDoProgramowania/pull/1/commits/181d2f9dbb054ce3dc2707766337e548f9f3a365)

Reply:
Gites